scenariolib/visit.go

```diff
@@ -134,11 +134,16 @@ func (v *Visit) ExecuteScenario(scenario Scenario, c *Config) error {
 		if v.WaitBetweenActions {
 			var timeToWait int
 			if c.TimeBetweenActions > 0 {
-				timeToWait = c.TimeBetweenActions
+				timeToWait = c.TimeBetweenActions //Random waiting time
+				WaitBetweenActions(timeToWait)
+			}else if c.TimeBetweenActions < 0 {
+				timeToWait = c.TimeBetweenActions*(-1)
```

Comment:
Missing spaces around `*`, missing space before `else`.

I find it weird to use a negative value to set a time to wait. Why not use another parameter?